fstest/fstest.go

```diff
@@ -47,7 +47,7 @@ var (
 	// ListRetries is the number of times to retry a listing to overcome eventual consistency
 	ListRetries = flag.Int("list-retries", 3, "Number or times to retry listing")
 	// MatchTestRemote matches the remote names used for testing
-	MatchTestRemote = regexp.MustCompile(`^rclone-test-[abcdefghijklmnopqrstuvwxyz0123456789]{24}$`)
+	MatchTestRemote = regexp.MustCompile(`^rclone-test-[abcdefghijklmnopqrstuvwxyz0123456789]{12}$`)
```

Comment:
This also needs to change

https://github.com/rclone/rclone/blob/0735f44f910c3bafe38a7e7e04b07ade48edc1b7/fstest/test_all/clean.go#L20

Though maybe that one should say `{12,24}` to clean the old and the new!

Reply:
Made this change.

Reply:
Thanks. I've cherry-picked this now.